File: ErdosRenyi.m
function G=ErdosRenyi(N,P)

%%
% With these two parameters, we can instantiate the graph. The 
% variable |G| is the adjacency matrix for the graph. However,
% the first step doesn't treat edges symmetrically. The last
% two operations fix this and yield a symmetric adjacency matrix.

rand('seed',100); % reseed so you get a similar picture
G = rand(N,N) < P;
G = triu(G,1);
G = G + G'; 

File: PythonCode/random_networks/erdos_renyi.py
import numpy as np


def ErdosRenyiNetwork(n, p, seed=None):
    if seed:
        np.random.seed(seed)
    G = np.random.rand(n, n) < p
    G = np.triu(G, 1)
    G = G + G.T
    return G
